docs/modules/operation/pages/bsm/edges.adoc

```diff
@@ -5,36 +5,35 @@ Edges map the alarm status monitoring with OpenNMS.
 
 The following types can be used:
 
-* Child Service: A reference to an existing business service on which to depend
-* IP Service: A convenient way to refer to the alarms that can be generated by a monitored IP service. This will automatically provided edges for the nodeLostService, interfaceDown, and nodeDown reductions keys of the specified service.
-* Reduction Key: A resolved reduction key used to refer to a specific alarm, e.g., generated by an SNMP trap or threshold violation
-* Application: A reference to an existing application. This will automatically provide edges for the nodeLostService, interfaceDown, and nodeDown reductions keys of the defined services of this application.
+Child Service:: A reference to an existing business service on which to depend.
+IP Service:: A convenient way to refer to the alarms that can be generated by a monitored IP service.
+This will automatically provide edges for the nodeLostService, interfaceDown, and nodeDown reductions keys of the specified service.
```

Comment:
```suggestion
This will automatically provide edges for the nodeLostService, interfaceDown, and nodeDown reduction keys of the specified service.
```